src/common/lib/client/realtimechannel.ts

```diff
@@ -630,6 +637,7 @@ class RealtimeChannel extends Channel {
       case actions.ATTACHED: {
         this.properties.attachSerial = message.channelSerial;
         this._mode = message.getMode();
+        console.log('Channel.onMessage ATTACHED, setting params to:', (message as any).params);
```

Comment:
Is this meant to be here? Should it be using the internal logging API if so?

Reply:
whoops, my bad, thanks